### DISCUSSION TITLE
Does the Wifibox currently support the Intel BE200 WiFi 7 card?

### DISCUSSION THREAD

**Discussion by ChomperT** [pgj/freebsd-wifibox#81] [General]
In September 2023 and November 2023, the firmware was updated for the following two components.
[iwlwifi-ma-b0-gf-a0-83.ucode](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/tree/iwlwifi-ma-b0-gf-a0-83.ucode)
[iwlwifi-ma-b0-gf-a0-86.ucode](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/tree/iwlwifi-ma-b0-gf-a0-86.ucode)

**Comment by pgj** [comment]
Is the question that whether the recent release of Wifibox includes these firmware files?

**Comment by pgj** [comment]
The `20230926` version of Wifibox/Alpine (the "firmware" of Wifibox) includes the `20230919` version of `linux-firmware`.  This version does not yet have the referenced files but `20231211`.  Therefore the answer is probably no.

I can update `linux-firmware` to `20231211` in the [development repository](https://github.com/pgj/freebsd-wifibox-port) if you wish to give it a try.  But it may also need a more up-to-date driver which means updating the Linux kernels too.  We will see.

**Reply by ChomperT** [reply]
I will try this version, thank you for your reply

**Comment by ChomperT** [comment]
I tried testing this version yesterday, and currently, it cannot recognize the BE200 network card properly.

**Reply by pgj** [reply]
Please try it again now with the `20240116` version of the `net/wifibox-alpine` port.  It includes the `20240115` version of `linux-firmware`.  Note that, by default, the port will use Linux 6.1 (LTS), but if it does not work, you can try to use Linux 6.5 (edge) by setting the `KERN_EDGE` option.

**Reply by ChomperT** [reply]
I tried this version and it works fine!

**Reply by pgj** [reply]
I am glad to hear that!  Was it enough to use the LTS kernel or did it work only with the edge?

**Reply by ChomperT** [reply]
edge kernel
